src/mtev_rest.c

```diff
@@ -315,10 +318,17 @@ mtev_http_get_handler(mtev_http_rest_closure_t *restc, mtev_boolean *migrate) {
     restc->fastpath = rule->handler;
     restc->closure = rule->closure;
     restc->aco_enabled = rule->aco_enabled;
-    if(rule->pool) {
+    *migrate = mtev_false;
+    if(!mtev_http_session_aco(restc->http_ctx)) {
```

Comment:
```suggestion
    if(mtev_http_session_aco(restc->http_ctx)) {
```
Shouldn't this be testing for true not false here ..?

Reply:
I don't think so. We can do migration until we go aco. After that, it is not okay. We should only do the migration in the case where we are not aco.

Reply:
Ah, you're correct .. This is for before.